javascript/javascript2/week1/danish.js

```diff
@@ -0,0 +1,27 @@
+function findNumberOfDanishLetters(str) {
+  let letterFrequency = {};
+  let totalOfLetters = 0;
+  const danishLetters = ["æ", "ø", "å"];
+  str = str.toLowerCase();
+  if (typeof str !== "string") {
```

Comment:
This is a great idea! But you should do it before calling `toLowerCase` to avoid an error when `str` is not a string:
<img width="323" alt="image" src="https://github.com/bhumikama/hyf-homework/assets/5467758/cb0de021-de30-49d3-9d72-18081f7cb889">